tofu/openadas2tofu/_read_files.py

```diff
@@ -228,7 +530,17 @@ def step03_read_all(
             msg = "\tLoading data from {}".format(pfe)
             print(msg)
         dout = func(pfe, dout=dout, **kwdargs)
-    return dout
+
+    if typ1 == 'adf15' and format_for_DataCollection is True:
```

Comment:
`and format_for_DataCollection` ?

Reply:
Do you mean:
* "Why `and format_for_DataCollection is True` instead of just `and format_for_DataCollection` ?"
or 
* "What does `format_for_DataCollection` stand for?"
What's the question?

Reply:
the first one, sorry

Reply:
No particular reason, actually maybe the second option is better, I can't decide really

Reply:
> I know it's not important but I'm just wondering: is there a reason that you didn't make the changes so that pep8speaks doesn't complain? just lack of time?

All changes that say `continuation line missing indentation or outdented` I cannot really change, it's due to line breaks that are necessary.
Others appeared recently and were not there last time I checked (sometimes pep8speak takes a while to update).
I'll fix them in my next PR, good point